README.md

```diff
@@ -28,8 +47,209 @@ You can also make categories for the posts and have account so will be able to f
 
 `> docker-compose up`
 
+## Tests
+
+Run tests with
+
+`> npm run test`
+
 ## Technologies used
 
 ### Language
 
 Application is written in typescript.
+
+## Live Demo
+
+Application has been deployed to cloud application platform called Heroku.  
+Live demo of this application can be found in:  
+https://epicbulletinboard.herokuapp.com/
```

Comment:
Maybe change this url to `https://epicbulletinboard.herokuapp.com/posts` so you instantly get some data?